packages/mdc-side-sheet/_mixins.scss

```diff
@@ -0,0 +1,214 @@
+  @include mdc-shape-radius($radius, $rtl-reflexive: true);
+}
+
+@mixin mdc-side-sheet-scrim-fill-color($color, $opacity: $mdc-side-sheet-modal-scrim-opacity) {
+  $value: rgba(mdc-theme-prop-value($color), $opacity);
+
+  ~ .mdc-side-sheet-scrim {
+    @include mdc-theme-prop(background-color, $value);
+  }
+}
+
+@mixin mdc-side-sheet-ink-color($color, $opacity: $mdc-side-sheet-ink-opacity) {
+  $value: rgba(mdc-theme-prop-value($color), $opacity);
+
+  @include mdc-theme-prop(color, $value);
+}
+
+@mixin mdc-side-sheet-surface-fill-color($color) {
+  $value: mdc-theme-prop-value($color);
+
+  @include mdc-theme-prop(background-color, $value);
+}
+
+@mixin mdc-side-sheet-surface-fill-color-accessible($color) {
+  $accessibleColor: mdc-theme-accessible-ink-color($color);
+
+  @include mdc-side-sheet-ink-color($accessibleColor);
+  @include mdc-side-sheet-surface-fill-color($color);
+}
+
+@mixin mdc-side-sheet-z-index($value) {
+  z-index: $value;
+}
+
+@mixin mdc-side-sheet-width($width, $side: trailing) {
+  width: $width;
+
+  &.mdc-side-sheet--open:not(.mdc-side-sheet--closing) ~ .mdc-side-sheet-app-content {
+    @if $side == leading {
+      @include mdc-rtl-reflexive-box(margin, left, $width);
+    } @else if $side == trailing {
+      @include mdc-rtl-reflexive-box(margin, right, $width);
+    } @else {
+      @error "mdc-side-sheet-width() mixin: $side must be 'leading' or 'trailing', but got '#{$side}'";
+    }
+  }
+}
+
+//
+// Private
+//
+
+@mixin mdc-side-sheet-stroke-right_($width) {
+  border-right-width: $width;
+  border-left-width: 0;
+  border-right-style: solid;
+  border-left-style: none;
+}
+
+@mixin mdc-side-sheet-stroke-left_($width) {
+  border-right-width: 0;
+  border-left-width: $width;
+  border-right-style: none;
+  border-left-style: solid;
+}
+
+@mixin mdc-side-sheet-stroke-width_($width, $side: trailing) {
+  @if $side == leading {
+    @include mdc-side-sheet-stroke-right_($width);
+
+    @include mdc-rtl {
+      @include mdc-side-sheet-stroke-left_($width);
+    }
+  } @else if $side == trailing {
+    @include mdc-side-sheet-stroke-left_($width);
+
+    @include mdc-rtl {
+      @include mdc-side-sheet-stroke-right_($width);
+    }
+  } @else {
+    @error "mdc-side-sheet-stroke_() mixin: $side must be 'leading' or 'trailing', but got '#{$side}'";
+  }
+}
+
+@mixin mdc-side-sheet-base_($side: trailing) {
+  $xFactor: if($side == leading, 1, -1);
+
+  order: if($side == leading, 0, 1);
+
+  &--animate {
+    transform: translateX(-100% * $xFactor);
+
+    @include mdc-rtl {
+      transform: translateX(100% * $xFactor);
+    }
+  }
+
+  &--opening {
+    transform: translateX(0);
+
+    // Required to level the specificity with animate class.
+    @include mdc-rtl {
+      transform: translateX(0);
+    }
+  }
+
+  &--closing {
+    transform: translateX(-100% * $xFactor);
+
+    @include mdc-rtl {
+      transform: translateX(100% * $xFactor);
+    }
+  }
+}
+
+@mixin mdc-side-sheet-scrim_($side: trailing) {
```

Comment:
I don't think the branching in this mixin is necessary since the scrim is 100% width.

Meanwhile, why aren't the rest of the scrim styles in here? They should all be shareable/common with drawer, right? The point of these is to avoid needing to repeat the same styles between mdc-side-sheet and mdc-drawer codebases. (I think this question also applies to other mixins here)